ml4h/models/model_factory.py

```diff
@@ -31,7 +30,9 @@
 
 BLOCK_CLASSES = {
     'conv_encode': ConvEncoderBlock,
+    'merge_conv_encode': ConvEncoderMergeBlock,
```

Comment:
Is this block an encoder or merge block -- do you give this to the arg `--encoder_blocks` or `--merge_blocks`? If it is for `encoder_blocks` then can you rename to `unet_conv_encode`?

Reply:
It's for `--merge-blocks`.